content/browser/renderer_host/brave_plugin_registry_impl.cc

```diff
@@ -0,0 +1,38 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/content/browser/renderer_host/brave_plugin_registry_impl.h"
+
+#include "base/strings/utf_string_conversions.h"
+
+namespace content {
+
+BravePluginRegistryImpl::BravePluginRegistryImpl(ResourceContext* resource_context)
+    : PluginRegistryImpl(resource_context) {}
+
+BravePluginRegistryImpl::~BravePluginRegistryImpl() {}
+
+void BravePluginRegistryImpl::GetPluginsComplete(GetPluginsCallback callback, 
+  std::vector<blink::mojom::PluginInfoPtr> plugins) {
+  std::vector<blink::mojom::PluginInfoPtr> filtered_plugins;
```

Comment:
this skips calling the superclass method. You need to pass the superclass method below in `GetPlugins`, but instead of binding `callback` directly, you should bind a BravePluginRegistry method that matches the signature of GetPluginsCallback, but with an extra param so you can prebind the actual callback

Reply:
forget it, this is correct